fineract-provider/build.gradle

```diff
@@ -274,10 +252,50 @@ bootJar {
     classifier = ''
 }
 
-bootWar {
-    enabled = false
+jib {
+    from {
+        // image = 'azul/zulu-openjdk-alpine:17-jre' // once we get rid of OpenJPA we could reduce space significantly
+        image = 'azul/zulu-openjdk-alpine:17'
+    }
+
+    to {
+        image = 'fineract'
+        tags = [
+            "${releaseVersion}",
+            'latest'
+        ]
+    }
+
+    container {
+        creationTime = 'USE_CURRENT_TIMESTAMP'
+        jvmFlags = [
+            '-Duser.home=/data',
+            '-Xmx1G',
+            '-Xms1G',
+            '-XshowSettings:vm',
+            '-XX:+UseContainerSupport',
+            '-XX:+UseStringDeduplication',
+            '-XX:MinRAMPercentage=25',
+            '-XX:MaxRAMPercentage=80'
+        ]
+        args = [
+            '-Dspring.profiles.active=basicauth',
+            '-Dfile.encoding=UTF-8',
+            '-Duser.timezone=UTC',
```

Comment:
At least previously the integration tests required timezone Asia/Kalkuta to pass... does this work?

Reply:
Well... we should override such test specific settings in the Docker Compose file now... UTC as a default makes sense I think... users need to override this one anyway in prod.
In terms of unit tests: I think less than a handful depend on the timezone.

Reply:
Yes - I didn't mean changing the default if it's possible for us to override it for the integration tests. Unfortunately I think there's around 20-30 tests that fail at different times of day (some before midnight, some after midnight etc) if the timezone is not set to Asia/Kalkuta

Reply:
Alright... I'll set this in the Docker Compose file, no problem.

Reply:
Done. The complete entry point is now available in the Docker Compose file. Serves also as an example for other configuration (e. g. mem reservation etc.)